testing/frr/frr.initd

```diff
@@ -0,0 +1,19 @@
+#!/sbin/openrc-run
+
+description="FRRouting"
+
+start() {
+        /usr/lib/frr/frr start
```

Comment:
The main executable in `/usr/lib`?!

Reply:
The user-facing executable is in /usr/bin/vtysh, these are system daemons, docs and other distros have them in /usr/lib/frr.  In frr, the user does not directly interact w/ the routing daemons...

/usr/lib/frr/frr is just a startup shell script.  Upstream didn't want yet another startup script, so we could re-write it and put it in aports, but that seems like a lot of work, esp since I'm not sure of what benefits we'd get back for that...